da/executor/dispersal.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package dispersal;
+
+message Blob {
```

Comment:
Ideally this should go into a separate file, where common DA protocol types are defined.